packages/cli/src/smoke-tests/smoke-tests.js

```diff
@@ -82,12 +82,14 @@ describe('smoke tests - setup will take some time', () => {
     fs.removeSync(context.workdir);
   });
 
-  it('package size should not change much', async () => {
+  it('package size should not change much', async function() {
```

Comment:
To access `this` in the test case, we have to use `function()` instead of an arrow function.